File: Selenium/Project/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("http://alchemy.hguy.co/crm")

    userName = driver.find_element(By.ID,"user_name")
    userName.send_keys("admin")
    
    password = driver.find_element(By.ID,"username_password")
    password.send_keys("pa$$w0rd")
	
    login = driver.find_element(By.ID,"bigbutton")
    login.click()

    activitiesMenu = driver.find_element(By.XPATH,"//*[@id=\"grouptab_3\"]")
    
    print(f"The text of activites menu is: {activitiesMenu.text}")
    assert activitiesMenu.is_displayed()

    driver.quit()

File: Selenium/Project/Activity6.java
package seleniumActivities;

import static org.testng.Assert.assertEquals;
import static org.testng.Assert.assertTrue;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity6 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
		driver = new FirefoxDriver();
		driver.get("http://alchemy.hguy.co/crm");
		WebElement userName = driver.findElement(By.id("user_name"));
		WebElement password = driver.findElement(By.id("username_password"));
		WebElement login = driver.findElement(By.id("bigbutton"));
		
		userName.sendKeys("admin");
		password.sendKeys("pa$$w0rd");
		login.click();
	}
	
	@Test
	public void activities() {
		WebElement activitiesMenu = driver.findElement(By.xpath("//*[@id=\"grouptab_3\"]"));
		System.out.println("The text of activites menu is: "+activitiesMenu.getText());
		assertTrue(activitiesMenu.isDisplayed());	
	}
	@AfterClass
	public void tearDown() {
		driver.quit();
	}
}
